frontend/src/components/AutoRotatingCarouselComponent.js

```diff
@@ -0,0 +1,69 @@
+import React, {useState} from "react";
+import {AutoRotatingCarousel, Slide} from "material-auto-rotating-carousel";
+import Button from "@material-ui/core/Button";
+
+import useMediaQuery from "@material-ui/core/useMediaQuery";
+import {yellow, blue, cyan} from "@material-ui/core/colors";
+
+const AutoRotatingCarouselModal = ({ handleOpen, setHandleOpen, isMobile }) => {
+    return (
+        <div>
```

Comment:
i think you can remove the div or do you use it for styling?

Reply:
done